db/forms/kpi/gbv_assessment.rb

```diff
@@ -0,0 +1,28 @@
+gbv_assessment_fields = [
+  Field.new({"name" => "completed_asssessment",
+    "mobile_visible" => false,
+    "type" => "tick_box",
+    "display_name_en" => "Completed assessment",
+    "editable" => false,
+    "disabled" => true
+  }),
+  Field.new({"name" => "approved_assessment",
+    "mobile_visible" => false,
+    "type" => "tick_box",
+    "display_name_en" => "Completed assessment, supervisor approved",
+    "editable" => false,
+    "disabled" => true
+  }),
+];
+
+FormSection.create_or_update_form_section({
+  unique_id: "gbv_assessment",
+  parent_form: "kpi",
```

Comment:
I'm struggling to understand why it is necessary to add a new group of KPI forms.  How do these forms/fields get populated?  How are they used?   Should they just be properties on the kpi model instead?